src/it/java/org/checkstyle/base/AbstractItModuleTestSupport.java

```diff
@@ -451,6 +481,200 @@ protected final void verify(Checker checker,
         checker.destroy();
     }
 
+    /**
+     * Performs verification of the file with the given file path against google config.
+     * It uses the specified list of modules to load them from google config for validation.
+     * And also uses the array of expected messages for verification.
+     *
+     * @param listOfModules list of modules to load from google config.
+     * @param filePath file path to verify.
+     * @throws Exception if exception occurs during verification process.
+     */
+    public final void verifyWithGoogleConfigParser(Map<String, String[]> listOfModules,
+           String filePath) throws Exception {
+        final DefaultConfiguration googleConfig = createGoogleConfig(listOfModules);
+        final TestInputConfiguration testInputConfiguration =
+                InlineConfigParser.parseGoogleConfig(filePath);
+        final List<String> actualViolations = getActualViolationsForFile(googleConfig, filePath);
+
+        verifyViolations(filePath, testInputConfiguration.getViolations(), actualViolations);
```

Comment:
>InlineConfigParser.parseGoogleConfig(filePath);

this is nonsense, we do not parse config out of Input.
please rename it to getViolationsFromIntegrationInput

Reply:
done

https://github.com/checkstyle/checkstyle/blob/a0f82691156c959fbb7dccd991c3e5c4f83022e1/src/it/java/org/checkstyle/base/AbstractItModuleTestSupport.java#L493-L497

https://github.com/checkstyle/checkstyle/blob/a0f82691156c959fbb7dccd991c3e5c4f83022e1/src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java#L221-L224